src/System.Data.DataSetExtensions/pkg/System.Data.DataSetExtensions.pkgproj

```diff
@@ -3,9 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Data.DataSetExtensions.csproj">
-      <SupportedFramework>netcoreapp2.0;net461;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>netcoreapp2.0;net45;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Data.DataSetExtensions.csproj" />
+    <InboxOnTargetFramework Include="net45">
```

Comment:
@joperezr looked at this with me and adding this creates an unnecessary spot for net45, we already have it in the rooted lib